tests/BadAttributes.py

```diff
@@ -52,11 +57,11 @@ def testBadAttributeTransformation(self):
         self.assertEqual(raisedexp.exception.actual_value, "foobar")
         self.assertEqual(raisedexp.exception.expected_type, str)
         self.assertEqual(
-            raisedexp.exception.transformation_exception.__class__, ValueError
+            raisedexp.exception.transformation_exception.__class__, ParserError
         )
```

Comment:
Doesn't the testing api have `assertIsInstance`?

Reply:
It does, this looks like poor legacy code. I didn't spot this, I just adjusted the expectation :face_with_spiral_eyes:.